package.json

```
@@ -17,7 +17,9 @@
     "test-cypress-headless": "cd integration-tests && node --max-old-space-size=4096 ../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome}",
     "cypress-merge": "mochawesome-merge ./integration-tests/screenshots/cypress_report*.json > ./integration-tests/screenshots/cypress.json",
     "cypress-generate": "marge -o ./integration-tests/screenshots/ -f cypress-report -t 'OpenShift Console Plugin Template Cypress Test Results' -p 'OpenShift Cypress Plugin Template Test Results' --showPassed false --assetsDir ./integration-tests/screenshots/cypress/assets ./integration-tests/screenshots/cypress.json",
-    "cypress-postreport": "yarn cypress-merge && yarn cypress-generate"
+    "cypress-postreport": "yarn cypress-merge && yarn cypress-generate",
+    "generate-types": "openapi-typescript openapi/spec.yml --output src/types/api.ts",
+    "start:dev": "npx nodemon"
```

Its not restarting the webpack server properly. 

Better to go with the one used in the [Kubevirt Plugin](https://github.com/kubevirt-ui/kubevirt-plugin/blob/4c03b2d7a68530ef9539d1f52a85a538da935c41/package.json#L176C5-L176C147)

```suggestion
 "dev": "NODE_ENV=development NODE_OPTIONS=--max-old-space-size=8192 rm -rf dist && yarn ts-node ./node_modules/.bin/webpack serve --progress"
```

---

Not needed to run this command - It can be run with this - `npx ts-node src/components/api-type-test.ts`
I am updating the PR to not use express and get the results from consoleFetch and display on the ui, so that there are no additional steps.